java-chassis-dependencies/default/pom.xml

```diff
@@ -1341,6 +1343,11 @@
         <artifactId>business-service-springmvc-archetype</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
```

Comment:
Add new libraries need update LICENCE file in ${HOME}/java-chassis-distributioni/release/src/LICENCE

Reply:
I had add 
"* nacos-client (com.alibaba.nacos:1.1.4 - https://github.com/alibaba/nacos)"
 in LICENSE file.